README.md

```diff
@@ -14,14 +14,9 @@ HOW TO USE IT
 
 [Use the source](http://github.com/bartaz/impress.js/blob/master/index.html), Luke ;)
 
-If you have no idea what I mean by that, or you just clicked that link above and got
-very confused by all these strange characters that got displayed on your screen,
-it's a sign, that impress.js is not for you.
+impress.js was created with developers in mind and some basic knowledge of html, css and javascript is a prerequisite to install and utilize it's features.
 
-Sorry.
-
-Fortunately there are some guys on GitHub that got quite excited with the idea of building
-editing tool for impress.js. Let's hope they will manage to do it.
+If you are a designer or novice developer and intent on using impress, there is a project on GitHub that aims to provide an editing tool for impress.js. It is still in development but we hope they will manage to make impress.js more accessible to all.
 
```

Comment:
Unfortunately this also changed the intent of the sentence. It is not just one project that aims to provide an editing tools, there are [more than one of them](https://github.com/impress/impress.js/wiki/Examples-and-demos/5d887507caa5cf534eab6713d9adb3a5e7662459). Can we focus in the first paragraph only though? Or this paragraph also causes trouble?